odoo/addons/base/data/res_country_data.xml

```diff
@@ -930,11 +975,13 @@
             <field name="address_format" eval="'%(street)s\n%(street2)s\n%(zip)s %(city)s\n%(country_name)s'" />
             <field name="currency_id" ref="CHF" />
             <field eval="423" name="phone_code" />
+            <field name="address_view_id" ref="base.view_partner_address"/>
         </record>
         <record id="lk" model="res.country">
             <field name="name">Sri Lanka</field>
             <field name="code">lk</field>
             <field file="base/static/img/country_flags/lk.png" name="image" type="base64" />
+            <field eval="'%(street)s\n%(street2)s\n%(city)s\n%(zip)s\n%(country_name)s'" name="address_format" />
```

Comment:
Add address_view_id = view_partner_address_street_city_zip_state_country for Sri Lanka